src/pages/Mypage/ProfilePage.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import styled from "styled-components";
+import ProfileInfoCard from "./components/ProfileInfoCard";
+import CardEstimate from "./components/CardEstimate";
+
+const ProfilePage = () => {
+  return (
+    <Container>
+      <section>
+        <FirstTitle>개인정보</FirstTitle>
+        <ProfileInfoBox>
+          {PROFILE_INFO_DATA.map(item => (
+            <ProfileInfoCard
+              key={item.id}
+              id={item.id}
+              title={item.title}
+              value={item.value}
+            />
```

Comment:
```suggestion
            <ProfileInfoCard
              key={item.id}
              {...item}
            />
```

props로 데이터 넘길 때 데이터에 있는 모든 값을 넘기고
해당 값이 같은 key로 넘길때 위와 같이 props를 전달할 수 있습니다.

Reply:
피드백 반영했습니다!